base_comment_template/migrations/12.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,62 @@
+import logging
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+
+    _logger.debug('Migrating res.partner comment_template_id')
+
+    cr.execute("""
+        SELECT column_name FROM information_schema.columns
+        WHERE table_name='res_partner'
+        AND column_name='comment_template_id';
+    """)
+
+    if not cr.fetchall():
+        _logger.debug('Column \'comment_template_id\' wasn\'t present')
+        return
+
+    cr.execute("SELECT id FROM res_company")
+    company_ids = [d[0] for d in cr.fetchall()]
+
+    cr.execute(
```

Comment:
This code seems a very good candidate for a function in https://github.com/OCA/openupgradelib called `convert_field_to_property` or similar.

Reply:
I created ticket https://github.com/OCA/openupgradelib/issues/147, I'll do that later when I have time

Reply:
Thanks